pkg/overlay/cache_test.go

```diff
@@ -285,12 +285,12 @@ func TestIsVetted(t *testing.T) {
 		},
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
 		var err error
-		satellite := planet.Satellites[0]
-		satellite.Audit.Service.Loop.Pause()
-		satellite.Repair.Checker.Loop.Pause()
-		service := satellite.Overlay.Service
+		peer := planet.Satellites[0]
```

Comment:
why peer instead of satellite?  satellite is more specific

Reply:
the name satellite conflicts with the package satellite name and it has caused issues in the past... Goland recommended the name `peer` but I'm open to other options

Reply:
I feel like peer is overly generic.  _sat_ works for me; I'm not sure if others would find that to be clear enough.

Reply:
Ok I like _sat_